homeworks/Vladimir.Karbivnychiy_VladimirSkarb/5-rxjs/app.js

```diff
@@ -0,0 +1,48 @@
+const { fromEvent } = window.rxjs;
+const {
+    pairwise,
+    distinctUntilChanged,
+    map,
+    throttleTime,
+    filter,
+} = window.rxjs.operators;
+const scrollEvent = fromEvent(window, 'scroll');
+const button = document.querySelector('[data-btn]');
+const firstHeader = document.querySelector('[data-first-header]');
+const thirdHeader = document.querySelector('[data-third-header]');
+const secondHeader = document.querySelector('[data-second-header]');
+
+scrollEvent.pipe(
+    map((event) => event.path[1].pageYOffset),
+    throttleTime(300),
+    pairwise(),
+    filter(([num1, num2]) => (num1 - num2) > 50 || (num2 - num1) > 50),
```

Comment:
Choose better names for the variables. (previousHeight, currentHeight)
Use Math.abs to avoid several checks